Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM debian:stable as builder
+
+# Let's install dependencies
+RUN apt-get update && apt-get --allow-unauthenticated -qq install qt5-default qtbase5-dev libqt5sql5-sqlite libssl-dev libmosquittopp-dev cmake git build-essential
+
+WORKDIR /build
+
+# Install Astarte Qt5 SDK
+RUN git clone https://github.com/astarte-platform/astarte-device-sdk-qt5.git && \
+	cd astarte-device-sdk-qt5 && \
+	mkdir build && \
+	cd build && \
+	cmake -DCMAKE_INSTALL_PREFIX=/usr .. && \
+	make && \
+	make install && \
+	cd -
+
+# Install Astarte Qt5 Stream Test
+RUN git clone https://github.com/astarte-platform/stream-qt5-test.git && \
+	cd stream-qt5-test && \
+	qmake . && \
+	make && \
+	cd -
+
+FROM debian:stable-slim
+
+# Install required dependencies
+RUN apt-get update && apt-get --allow-unauthenticated -qq install libmosquittopp1 libqt5network5 libqt5sql5-sqlite libssl1.1
+
+# Add our fresh binaries
+COPY --from=builder /usr/lib/libAstarteDeviceSDKQt5.so.0* /usr/lib/
+COPY --from=builder /usr/bin/astarte-validate-interface /usr/bin/
+COPY --from=builder /usr/share/hyperdrive /usr/share/hyperdrive
+COPY --from=builder /build/stream-qt5-test/stream-qt5-test /usr/bin/
```

Comment:
You also need to copy the `interfaces` folder of `stream-qt5-test` in the directory where `stream-qt5-test` will be executed
```
COPY --from=builder /build/stream-qt5-test/interfaces /
```

Reply:
fixed copying locally